connector/src/main/java/com/google/cloud/spark/bigquery/SparkBigQueryConfig.java

```diff
@@ -88,7 +87,7 @@
   com.google.common.base.Optional<String> partitionField = empty();
   Long partitionExpirationMs = null;
   com.google.common.base.Optional<Boolean> partitionRequireFilter = empty();
-  com.google.common.base.Optional<String> partitionType = empty();
+  com.google.common.base.Optional<TimePartitioning.Type> partitionType = empty();
```

Comment:
Is TimePartitioning the only type of partitioning we will use now?

Reply:
Yes.